modules/installation-aws-upi-bootstrap.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws_upi/installing-aws-upi.adoc
+
+[id="installation-aws-upi-bootstrap-{context}"]
+= Initializing the bootstrap node on AWS UPI
+
+After you create all of the required infrastructure in Amazon Web Services (AWS),
+you can install the cluster.
+
+.Prerequisites
+
+* Configure an AWS account.
+* Generate the Ignition config files for your cluster.
+* Create and configure a VPC and assocated subnets in AWS.
+* Create and configure DNS, load balancers, and listeners in AWS.
+* Create control plane and compute roles.
+* Create the bootstrap machine.
+* Create the control plane machines.
+* If you plan to manually manage the worker machines, create the worker machines.
+
+.Procedure
+
+. Change to the directory that contains the installation program and run the
+following command:
++
+----
+$ ./openshift-install wait-for bootstrap-complete --dir=<installation-directory> <1>
```

Comment:
Incorrect command...
`$ bin/openshift-install wait-for bootstrap-complete --dir=/tmp/openshift/mycluster
FATA[0001] Error executing openshift-install: unknown command "wait-for" for "openshift-install"`

Maybe it should be...
`bin/openshift-install user-provided-infrastructure bootstrap-complete --dir=/tmp/openshift/mycluster`

Reply:
Your installer is too old, it's `wait-for` now.

Reply:
Try [beta4](https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.1.0-rc.0/).